runatlantis.io/blog.md

```diff
@@ -11,13 +11,13 @@ We are thrilled to have you here! Our blog is a collection of insightful article
 
 We have a rich history of blog posts dating back to 2017-2019. Here are some of our popular posts:
 
-- [4 Reasons To Try HashiCorp's (New) Free Terraform Remote State Storage](/blog/2019/4-reasons-to-try-hashicorps-new-free-terraform-remote-state-storage)
-- [I'm Joining HashiCorp!](/blog/2018/joining-hashicorp)
-- [Putting The Dev Into DevOps: Why Your Developers Should Write Terraform Too](/blog/2018/putting-the-dev-into-devops-why-your-developers-should-write-terraform-too)
-- [Atlantis 0.4.4 Now Supports Bitbucket](/blog/2018/atlantis-0-4-4-now-supports-bitbucket)
-- [Terraform And The Dangers Of Applying Locally](/blog/2018/terraform-and-the-dangers-of-applying-locally)
-- [Hosting Our Static Site over SSL with S3, ACM, CloudFront and Terraform](/blog/2018/hosting-our-static-site-over-ssl-with-s3-acm-cloudfront-and-terraform)
-- [Introducing Atlantis](/blog/2017/introducing-atlantis)
+- [4 Reasons To Try HashiCorp's (New) Free Terraform Remote State Storage](blog/2019/4-reasons-to-try-hashicorps-new-free-terraform-remote-state-storage.md)
```

Comment:
I thought you dont need to suffix `.md`

Reply:
As per the PR description, these changes are to allow the website link check to be able to be run on the source files, rather than needing to run the scan on a website build.